### PR TITLE
Show diffs introduced by running pre-commit (especially black)

### PR DESCRIPTION
**Pull request by jepler**

### CODE REVIEW

File: build/action.yml [hunk 1]
```diff
@@ -9,6 +9,12 @@ inputs:
     description: 'The version of Python to use in the CI'
     required: true
     default: '3.x'
+  package-prefix:
+    description: |
+      The prefix (or name) of your pacakge (if applicable) to use
```
Comment (tekktrik) [inline build/action.yml]: Good additional argument, just noticed a typo for "package" which is my doing, sorry!

**Review by tekktrik — DISMISSED** [review]
Good change for pre-commit, as well as the package prefix fix (fix fix)!

Just one typo but otherwise good to go!

**Comment by tekktrik** [comment]
I do know the failure help CI isn't working as intended still so I'll patch that up in its repo, and is outside the scope of this PR.

**Review by tekktrik — APPROVED** [review]
Thanks!  Went ahead and fixed the merge conflicts as well as the typos.

**Comment by jepler** [comment]
Thanks @tekktrik !

**Pull request merged**